app/controllers/comments_controller.rb

```diff
@@ -12,8 +15,37 @@ def create
             render :new, status: :unprocessable_entity
         end
     end
+
+    # Sends a CSV file containing an export of all visible comments of visible posts, ordered 
+    # ascendingly by the time of the comments' creation, with three columns: 
+    # "post title", "comment author", "comment body"
+    def export
+        current_datetime_string = DateTime.now.utc.strftime("%Y-%d-%m_%H-%M-%S")
```

Comment:
```suggestion
        current_datetime_string = DateTime.now.strftime("%Y-%d-%m_%H-%M-%S")
```
Perhaps it makes more sense to use the local server time. What do you think?